docs/crd.md

```diff
@@ -151,6 +152,8 @@ FlinkCluster
       * **pullSecrets** (optional): Secrets for image pull.
     * **batchSchedulerName** (optional): BatchSchedulerName specifies the batch scheduler name for JobManager, TaskManager.
       If empty, no batch scheduling is enabled.
+    * **serviceAccountName** (optional): serviceAccountName specifies the service account name(which must already exist in the namespace) for JobManager, TaskManager.
```

Comment:
the name of the service account (which must...). If empty, the default service account in the namespace will be used.

Reply:
fixed